tools/explorer/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ include(ExternalProject)
 set(TT_EXPLORER_SCRIPT ${CMAKE_CURRENT_SOURCE_DIR}/run.py)
 set(TTMLIR_BUILD_BIN_DIR ${TTMLIR_BINARY_DIR}/bin)
 
-set(MODEL_EXPLORER_VERSION "d0b53c3b7049fd41ea1caff193706272c399fac9")
+set(MODEL_EXPLORER_VERSION "50d3a757c990d30c69d0f9f544b66b0e6af89f81")
```

Comment:
I can't seem to build with this commit:
```
Cloning into 'model-explorer'...
remote: Enumerating objects: 4059, done.
remote: Counting objects: 100% (836/836), done.
remote: Compressing objects: 100% (349/349), done.
remote: Total 4059 (delta 645), reused 487 (delta 487), pack-reused 3223 (from 2)
Receiving objects: 100% (4059/4059), 10.03 MiB | 2.97 MiB/s, done.
Resolving deltas: 100% (2910/2910), done.
fatal: reference is not a tree: 50d3a757c990d30c69d0f9f544b66b0e6af89f81
CMake Error at /localdev/odjuricic/repos/tt-mlir/tools/explorer/model-explorer/tmp/model-explorer-gitclone.cmake:61 (message):
  Failed to checkout tag: '50d3a757c990d30c69d0f9f544b66b0e6af89f81'
```

Probably due to some rebase / squash / merge.

Should we just use tip of main now?

Reply:
Yeah let me change this to tip.